news/_posts/2021-07-06-responsivebo.md

```diff
@@ -0,0 +1,132 @@
+--- 
+layout: post
+title:  "Towards a more mobile-friendly back office"
+subtitle: "Quick wins of the responsive back office"
+date:   2021-07-06 09:00:00
+authors: [ScottCosio]
+icon: icon-mobile
+tags:
+ - 1.7.8
+ - version
+ - minor
+ - product
+ - ux
+---
+
+Until now, PrestaShop has always relied on a desktop-first responsive design strategy. Indeed, there are few uses cases on mobile. But you can't prevent users from checking their store's back office on their mobile!
+
+## Why work on responsive design?
+
+Today, there are some issues that affect the user experience on mobile device:
+
+- Difficulties in accessing important store informations
+- Hidden contents because of components overlaying others
+- Text readability issues
+- Bugs and lack of consistency on display
+
+All of these issues cause a bad reputation for the PrestaShop open source project.
```

Comment:
Are these issues still relevant today? Maybe you should write this paragraph in the past tense.  (+ add an "s" to device, or keep only "mobile" and remove the "s" from "information" (it's uncountable ☺️) 

You can also remove "the" before "user experience". ☺️ 

**Until now, they were** some issues that **affected** user experience on **mobile**. 

- Difficulties in accessing important store **information**
- Hidden contents because of components overlaying others
- Text readability issues
- Bugs and lack of consistency on display

All of these issues cause**d** a bad reputation for the PrestaShop open source project.